proto/plugin/plugin.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package helperPlugin;
+option go_package = "../../pkg/helperPlugin";
+
+message Empty {}
+
+message ConfigsRequest {
+    map<string,string> configs = 1;
```

Comment:
what this configs are? 
What data is the Plugin getting in order to understand where to find certs, or what kind of cert was received?
and how is each plugin configured? is it possible to provide data to this plugin?